Add mising header found with musl

Fixes
../../sandbox/linux/services/credentials.h:41:25: error: unknown type name 'uid_t'; did you mean 'pid_t'?
  static bool GetRESIds(uid_t* resuid, gid_t* resgid);
                        ^~~~~
                        pid_t

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>

---
 sandbox/linux/services/credentials.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/sandbox/linux/services/credentials.h b/sandbox/linux/services/credentials.h
index 17ff667cfd..66d4afbff1 100644
--- a/sandbox/linux/services/credentials.h
+++ b/sandbox/linux/services/credentials.h
@@ -14,6 +14,8 @@
 #include <string>
 #include <vector>
 
+#include <sys/types.h>
+
 #include "sandbox/linux/system_headers/capability.h"
 #include "sandbox/sandbox_export.h"
 
